Support for Angular 15

**Support for Angular 15.**
 npm install mit  angular 15 and latest ngx-vcard 4.1.14 doesn't work without --legacy-peer-deps 

**Bump version in package.json?**
 
Maybe just bump dependency in package.json and see if it works :-)

```
"peerDependencies": {
    "@angular/common": ">=5.x <15.0.0",
    "@angular/core": ">=5.x <15.0.0"
  },

```
   

**Alternative**
Using
` npm install --legacy-peer-deps `

helps but feels dirty :-)

**npm install  output**
                                                            
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: ngx-vcard@4.1.14
npm ERR! Found: @angular/common@15.0.4
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~15.0.4" from the root project
npm ERR!   peer @angular/common@"^15.0.0 || ^16.0.0" from @angular/cdk@15.0.3
npm ERR!   node_modules/@angular/cdk
npm ERR!     @angular/cdk@"^15.0.2" from the root project
npm ERR!   9 more (@angular/forms, @angular/platform-browser, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@">=5.x <15.0.0" from ngx-vcard@4.1.14
npm ERR! node_modules/ngx-vcard
npm ERR!   ngx-vcard@"^4.1.14" from the root project
```

our dependencies 

 ```
"@angular/animations": "~15.0.3",
    "@angular/cdk": "^15.0.2",
    "@angular/common": "~15.0.4",
    "@angular/compiler": "~15.0.3",
    "@angular/core": "~15.0.3",
    "@angular/forms": "~15.0.3",
    "@angular/platform-browser": "~15.0.3",
    "@angular/platform-browser-dynamic": "~15.0.3",
    "@angular/router": "~15.0.3",
  ..
    "ngx-vcard": "^4.1.14",
```

--- a/projects/ngx-vcard/package.json
+++ b/projects/ngx-vcard/package.json
@@ -24,7 +24,7 @@
     "tslib": "2.3.1"
   },
   "peerDependencies": {
-    "@angular/common": ">=5.x <15.0.0",
-    "@angular/core": ">=5.x <15.0.0"
+    "@angular/common": ">=5.x <16.0.0",
+    "@angular/core": ">=5.x <16.0.0"
   }
 }

